README.md

```diff
@@ -10,18 +10,40 @@ The humble beginnings of a Nim library similar to web3.[js|py]
 ## Installation
 
 You can install the developement version of the library through nimble with the following command
+
 ```
 nimble install https://github.com/status-im/nim-web3@#master
 ```
 
+## Development
+
+You should first run `./simulator.sh` which runs `ganache-cli`
+
+This creates a local simulated Ethereum network on your local machine and the tests will use this for their E2E processing
+
+### Interaction with nimbus-eth2
+
+This repo relies heavily on parts of the `nimbus-eth2` repo.
+In order to work properly here you need to `source /nimbus-eth2/env.sh`
```

Comment:
This comment is a bit out of place here. For an outsider, it would be a bit confusing and it applies equally well for all Status libraries in the vendor folder.

If such guidance doesn't already exist in the [Status Nim style guide](https://status-im.github.io/nim-style-guide/), it can be added there to explain our usage of the vendor folder.